### PR TITLE
Generate test code for receiver methods

### PR DESCRIPTION
**Pull request by cipepser**
This fix a regular expression which captures function name, to be able to generate test code for receiver methods.

### CODE REVIEW

File: spec/main-spec.js [hunk 1]
```diff
@@ -100,6 +100,7 @@ describe('gotests', () => {
       let text = 'package main' + nl + nl + 'func main()  {' + nl + '}' + nl
       text += 'func ReadConfigFile(filePath string) ([]string, error) {' + nl + '}'
       text += 'func  Strangely_named-Function  ( filePath string ) ( []string,error )  {' + nl + '}'
+      text += 'func(t *T) FuncWithReciever(a int) int {' + nl + '}'
```
Comment (abulimov) [inline spec/main-spec.js:103]: This just adds some text to test input, to actually test if regexp works for it you need to add `expect(functions).toContain('FuncWithReciever')` somewhere around line 114.

**Review by abulimov — COMMENTED** [review]
Thank you! All looks good except for missing check in test. Can you please address my comment? Once it's done I'll be happy to merge this and release new plugin version.

**Comment by cipepser** [comment]
I've fixed your pointing.

https://github.com/cipepser/atom-gotests/blob/work/spec/main-spec.js#L118

**Comment by abulimov** [comment]
Thanks, pushed new package with you fix.

**Pull request merged**